app/services/partners/family_request_create_service.rb

```diff
@@ -1,16 +1,16 @@
+# This service object is meant to transform a family request
+# into the correct parameters that would be expected a regular
+# Partners::ItemRequest. It will create the relevant
+# organization request (i.e. ::Request), the Partners::Request,
+# the Partners::ItemRequest(s), and Partners::ChildItemRequest(s)
 module Partners
   class FamilyRequestCreateService
```

Comment:
Based on our slack convo, I ended up copying the existing `FamilyRequestCreateService` to `app/services/partners/individual_request_create_service.rb` because I thought they'd diverge quite a bit, but looking at it now it's honestly pretty similar. I wonder if I should just DRY it up again

Reply:
I tried it out in this PR https://github.com/rubyforgood/diaper/pull/2278#issuecomment-817207001

I could go either way. the main difference now seems to just be the comments argument. And I had to add children inside `Partners::RequestCreateService` so it would create the associations automatically

Reply:
Closing in favor of https://github.com/rubyforgood/diaper/pull/2278#pullrequestreview-633004128